src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorDataSourceBundlerUtilsUnitTest.java

```diff
@@ -0,0 +1,99 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+
+/**
+ * A unit test suite for the {@link FuncotatorDataSourceBundlerUtils} class.
+ * Created by Hailey on 8/5/21.
+ */
+public class FuncotatorDataSourceBundlerUtilsUnitTest extends CommandLineProgramTest {
+
+    //==================================================================================================================
+    // Data Providers:
+
+    @DataProvider
+    Object[][] provideForTestGetDatasourceBaseName() {
+        return new Object[][] {
+                { FuncotatorDataSourceBundler.OrganismKingdom.PLANTS, "actinidia_chinensis", "Actinidia_chinensis.Red5_PS1_1.69.0.51" }
+        };
+    }
+
+    @DataProvider
+    Object[][] provideForTestGetFastaFileName() {
+        return new Object[][] {
+                { FuncotatorDataSourceBundler.OrganismKingdom.PLANTS, "actinidia_chinensis", "Actinidia_chinensis.Red5_PS1_1.69.0.cdna.all" }
+        };
+    }
+
+    @DataProvider
+    Object[][] provideForTestBuildMapWrong() {
+        return new Object[][]{
+                {
+                        FuncotatorDataSourceBundler.OrganismKingdom.PLANTS,
+                        "Aphanomyces astaci GCA_002197585.2"
+                }
+        };
+    }
+
+    @DataProvider
+    Object[][] provideForTestExtractGtfGz() {
+        return new Object[][] {
+            {
+                new File(largeFileTestDir + "funcotator/Acyrthosiphon_pisum.Acyr_2.0.51.PARTIAL.gtf.gz").getAbsolutePath(),
+                new File(largeFileTestDir + "funcotator/Acyrthosiphon_pisum.Acyr_2.0.51.PARTIAL.gtf").getAbsolutePath(),
+            },
+        };
+    }
+
+    //==================================================================================================================
+    // Tests:
+
+    @Test(dataProvider = "provideForTestGetDatasourceBaseName")
+    void testGetDatasourceBaseName(final FuncotatorDataSourceBundler.OrganismKingdom kingdom, final String speciesName, final String expectedBaseName) {
+
+        Assert.assertEquals(
+                expectedBaseName,
+                FuncotatorDataSourceBundlerUtils.getDatasourceBaseName(kingdom, speciesName)
```

Comment:
the actual value comes first, and the expected value second

Reply:
👍